modules/labs-react/expandable/stories/Expandable.stories.mdx

```diff
@@ -0,0 +1,131 @@
+import {Meta} from '@storybook/addon-docs';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Expandable} from '@workday/canvas-kit-labs-react/expandable';
+
+import {StartIcon} from './examples/StartIcon';
+import {EndIcon} from './examples/EndIcon';
+import {Avatar} from './examples/Avatar';
+import {Depth} from './examples/Depth';
+import {RTL} from './examples/RTL';
+import {LongTitle} from './examples/LongTitle';
+
+<Meta title="Labs/Expandable/React" component={Expandable} />
+
+# Canvas Kit Expandable
+
+`Expandable` is a compound component that creates a header to expand or collapse related content.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-labs-react
+```
+
+## Usage
+
+### Start Icon
+
+<ExampleCodeBlock code={StartIcon} />
+
+### End Icon
+
+<ExampleCodeBlock code={EndIcon} />
+
+### With Avatar
+
+<ExampleCodeBlock code={Avatar} />
+
+### Right to Left (RTL)
+
+<ExampleCodeBlock code={RTL} />
+
+### Depth
+
+<ExampleCodeBlock code={Depth} />
+
+### Title Wrap
+
+<ExampleCodeBlock code={LongTitle} />
```

Comment:
Probably adding a little bit more information about examples will be helpful in understanding them.